geometry_msgs/Pose2D Target
---
float64 Stiffness